<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BF_AnalizeForm</class>
 <widget class="QWidget" name="BF_AnalizeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>597</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Анализ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../analysis_resources.qrc">
    <normaloff>:/analysisplugin/img/plugin.png</normaloff>:/analysisplugin/img/plugin.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="4">
    <widget class="QPushButton" name="editButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Редактировать элемент требования</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/raznoe31/raznoe31/48x48/42.png</normaloff>:/images/raznoe31/raznoe31/48x48/42.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="updateButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Обновить все  требования</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/raznoe31/raznoe31/48x48/48.png</normaloff>:/images/raznoe31/raznoe31/48x48/48.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="deleteButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Удалить элемент требования</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/raznoe31/raznoe31/48x48/118.png</normaloff>:/images/raznoe31/raznoe31/48x48/118.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="onEditorButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Открыть редактор</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/raznoe31/raznoe31/48x48/13.png</normaloff>:/images/raznoe31/raznoe31/48x48/13.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QTabWidget" name="requirementsTab">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Видение</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="visionListWidget">
         <item>
          <property name="text">
           <string>Видение системы</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Предметная область</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Пользовательские требования</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QListWidget" name="userRequirementListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Функциональные требования</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="funcRequirementListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Нефункциональные требования</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="nofuncRequirementListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Общие требования</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="commonRequirementListWidget">
         <item>
          <property name="text">
           <string>Бизнес-правила</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Системные требования</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования к документированию</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования к дизаину ГИП</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования к безопасности и надежности</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования к производительности</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования к устойчивости к сбоям</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования к эксплуатации и персоналу</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования к внутреннему состоянию</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования к отладке</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования использования ПО</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования к аппаратуре</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Требования к стороннему ПО</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="addButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Добавить элемент требования</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../klever/core/frameWork/resources/framework_resources.qrc">
       <normaloff>:/images/raznoe31/raznoe31/48x48/112.png</normaloff>:/images/raznoe31/raznoe31/48x48/112.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../klever/core/frameWork/resources/framework_resources.qrc"/>
  <include location="../../analysis_resources.qrc"/>
 </resources>
 <connections/>
</ui>
